docs/resources/application.md

```diff
@@ -29,28 +29,56 @@ Axual Terraform Provider only supports Custom Application Type. Connector Applic
 ## Example Usage
 
 ```terraform
-resource "axual_application" "gitops_test_application_2" {
-  name    = "gitops_test_application_2"
+resource "axual_application" "dev_dashboard" {
+  name    = "DeveloperDashboard"
   application_type     = "Custom"
-  short_name = "gitops_test_application_2"
-  application_id = "axual.gitops.test2"
-  owners = axual_group.gitops_test.id
+  short_name = "dev_dash"
+  application_id = "io.axual.devs.dashboard"
+  owners = axual_group.developers.id
   type = "Java"
   visibility = "Public"
-  description = "Axual's Test Application for Gitops 2"
-  depends_on = [axual_stream_config.gitops_test_stream_config_2, axual_stream.gitops_test_stream2]
+  description = "Dashboard with crucial information for Developers"
 }
 
-resource "axual_application" "gitops_test_application_3" {
-  name    = "gitops_test_application_3"
+resource "axual_application" "log_scraper" {
+  name    = "LogScraper"
   application_type     = "Custom"
-  short_name = "gitops_test_application_3"
-  application_id = "axual.gitops.test3"
-  owners = axual_group.gitops_test.id
+  short_name = "log_scraper"
+  application_id = "io.axual.gitops.scraper1"
+  owners = axual_group.developers.id
   type = "Java"
   visibility = "Public"
-  description = "Axual's Test Application for Gitops 3"
-  depends_on = [axual_stream_config.gitops_test_stream_config_3, axual_stream.gitops_test_stream2]
+  description = "Axual's Test Application for finding all Logs for developers"
+}
+
+output "dashboard_id" {
```

Comment:
1. That's not true. The AWS provider has plenty of outputs. https://registry.terraform.io/providers/hashicorp/aws/latest/docs
2. I added them because some users were having trouble with outputs.
3. I think these examples give a good picture of what outputs to the user.

Reply:
It is up to the user to include the output, so perhaps we can add a comment "Include `output` if you want to have detailed information, e.g. for debugging purposes, as you can see below."?

Reply:
Not sure where to put this. It's a bit redundant putting it on every resource page. I don't think we need to document this further.